grid/crm/track-contact-event/profile.supr

```diff
@@ -0,0 +1,78 @@
+"""
+Track Event for Contact
+Track events that are specific for a single contact.
+"""
+name = "crm/track-contact-event"
+version = "1.0.0"
+
+"""
+Track Contact Event
+Sends an event data to CRM for a specific contact
+"""
+usecase TrackContactEvent unsafe {
+  input {
+    contactId!
```

Comment:
Would be helpful to document this property, ie. should I use an email or some provider-specific ID?